src/publish-rc/index.js

```diff
@@ -21,7 +21,7 @@ async function publishRc (opts) {
   await exec('git', ['fetch'])
 
   console.info(`Checking out branch ${opts.branch}`) // eslint-disable-line no-console
-  await exec('git', ['checkout', opts.branch])
+  await exec('git', ['checkout', '--track', `origin/${opts.branch}`])
```

Comment:
Maybe make this an option you pass in that defaults to `'origin'` instead of hard coding it?

Reply:
BTW, it's hard coded later in this function already for pushing...would be good to get this fix in and add the feature later?